stdlib/bigarray.mli

```diff
@@ -16,13 +16,42 @@
 (** Large, multi-dimensional, numerical arrays.
 
    This module implements multi-dimensional arrays of integers and
-   floating-point numbers, thereafter referred to as 'big arrays'.
+   floating-point numbers, thereafter referred to as 'big arrays',
+   to distinguish them from the standard OCaml arrays described in
+   {!module:Array}.
+
    The implementation allows efficient sharing of large numerical
    arrays between OCaml code and C or Fortran numerical libraries.
 
-   Concerning the naming conventions, users of this module are encouraged
-   to do [open Bigarray] in their source, then refer to array types and
-   operations via short dot notation, e.g. [Array1.t] or [Array2.sub].
+   The main differences between 'big arrays' and standard OCaml
+   arrays are as follows:
+   - Big arrays are not limited in size, unlike OCaml arrays.
+     (Normal float arrays are limited to 2,097,151 elements on a 32-bit
+     platform, and normal arrays of other types to 4,1943,03 elements.)
+   - Big arrays are multi-dimensional.  Any number of dimensions
+     between 1 and 16 is supported.  In contrast, OCaml arrays
```

Comment:
It is probably "between 0 and 16".

Reply:
Is it actually 0? Note I took this from previous documentation.

Reply:
I guess the documentation was not updated when `Array0`
was introduced -- see [here](https://github.com/ocaml/ocaml/blob/trunk/stdlib/bigarray.mli#L448).

Reply:
I will make this fix and push it.

Reply:
Pushed.